pvlib/solarposition.py

```diff
@@ -404,8 +406,12 @@ def sun_rise_set_transit_spa(times, latitude, longitude, how='numpy',
 
     Parameters
     ----------
-    times : pandas.DatetimeIndex
+    date : pandas.DatetimeIndex
```

Comment:
How did this name change creep into this PR?

Reply:
https://github.com/pvlib/pvlib-python/issues/2231#issuecomment-2389814884

Reply:
I changed my mind about waiting for feedback in the issue just to show another example, finally choose between each admonition type and test whether any decorator argument persisted between different uses.

I can revert back that change, sorry.

I considered this one should be renamed to `date` rather than `time`. In fact, the inner function being wrapped uses `dates` as the parameter name.

Reply:
Resolved, feel free to leave a review :D